docs/topics/iotea-protocol-adapter.md

```diff
@@ -0,0 +1,63 @@
+# IoTea Protocol Adapter Guide
+
+## Introduction
+
+The internal and external communication in IoT Event Analytics is abstracted using a provided Protocol Adapter. We are shipping IoT Event Analytics with an MQTT Protocol Adapter Implementation, but you can write your own adapter using a simple Interface and a program language specific Loading mechanism i.e. modules in NodeJS and packages in Python.
```

Comment:
idea: we could link the mqtt protocol adapter implementation that you can have a look to it if you are interested (as a developer)

Reply:
done